json-endpoints.php

```diff
@@ -2006,6 +2006,35 @@
 ) );
 
 new WPCOM_JSON_API_Site_User_Endpoint( array(
+	'description' => 'Get details of a user of a site by login.',
+	'group'       => '__do_not_document', //'users'
+	'stat'        => 'sites:1:user',
```

Comment:
Should we add an additional ID to the end here to distinguish this endpoint's stats?